docs/routing/special_routes.md

```diff
@@ -0,0 +1,60 @@
+# Special routes
+
+To see special routes, use the `Show special` switcher at the top of the routes page
+
+It is used to render errors, if an application with the `Primary` or `Essential` type is failed on the page, for example, when ILC will not find a match between the address bar and `Route`, `Template` will not be specified in the route chain, etc., ILC will use `Special route` or simple template with an error, depending on the situation:
+
+## '404' Error
+
+![ILC registry special routes switcher](../assets/routes/special-routes-switcher.png)
+
+- This error can appear in two cases:
+  1. If there is no route configured in Registry for requested URL.
+  2. The app fails to find the requested resource by it's ID (this rule is only for **primary** apps). More details about it [here](../global_errors_handling.md#404-error-not-found)
+
+- When using `Special route` all standard routes are ignored, including `*` with any `Order pos`. If you want something else to be displayed on the page besides the error (header, footer, etc.), you need to add them to the [Slot](./route_configuration_options.md#slot-configuration) as in the screenshot below (This is necessary because standard routes are ignored).
```

Comment:
This statement isn't clear. There is no way to "use" special route. 

It would be correct to say: Special routes disregard all of the previously matched routes with "next: true" specified. It means that you always need to specify all necessary slots & template here and shouldn't rely on the global wildcard routes.